packs/image/constants.ts

```diff
@@ -0,0 +1 @@
+export const PATH = "/live/invoke/deco-sites/std/loaders/x/image.ts";
```

Comment:
should it be `KEY` instead?
```suggestion
export const KEY = "/live/invoke/deco-sites/std/loaders/x/image.ts";
```